instructions.md

```diff
@@ -66,6 +68,9 @@ To clean up Kubernetes objects created by Sonobuoy, run:
 sonobuoy delete
 ```
 
+### Hydrophone
+Refer to the README.md [https://github.com/kubernetes-sigs/hydrophone/blob/main/README.md] for detailed instructions
```

Comment:
I'd suggest 

```
Refer to the [hydrophone docs](https://github.com/kubernetes-sigs/hydrophone/blob/main/README.md) for detailed instructions
```

to keep consistent with other links

Reply:
fized